### PR TITLE
Feature/update about pg

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: studio/schemas/documents/leaderCard.js [hunk 1]
```diff
@@ -18,5 +18,10 @@ export default {
             title: 'Image',
             type: 'mainImage',
         },
+        {
+            name: 'linkedinURL',
+            title: 'LinkedIn URL',
+            type: 'url',
+        },
```
Comment (asasmith) [inline studio/schemas/documents/leaderCard.js:25]: What is this for? Are they displaying only their linkedin url?

File: web/src/pages/about.js [hunk 2]
```diff
@@ -21,6 +21,22 @@ export const query = graphql`
...
+            leadershipSection {
+                leadership {
+                    name
+                    title
+                    image {
+                        asset {
+                            gatsbyImageData
+                        }
+                    }
+                    linkedinURL
```
Comment (asasmith) [inline web/src/pages/about.js:37]: I don't see where this is used in the about page.

File: web/src/pages/capabilities.js [hunk 3]
```diff
@@ -104,32 +110,16 @@ export default function CapabilitiesPage({ data }) {
...
-        },
-    ];
     return (
         <Layout>
             <Hero image={heroImage} title={pageTitle} />
             <ImageWithText data={capabilitiesAndServices} />
             <BgImagewithCard data={services} hasButton isSecondaryDark />
             <BgImagewithCard data={fleet} isRight hasButton isSecondaryDark />
-            <CertificationSection certifications={certifications} />
+            <CertificationSection certifications={nodes} />
```
Comment (asasmith) [inline web/src/pages/capabilities.js:122]: Does `nodes` work here?

File: studio/schemas/documents/leaderCard.js (hunk 1 above)
Comment (kcbreuil) [reply studio/schemas/documents/leaderCard.js:25]: @asasmith I thought we wanted to link the card to their linkedin. let me confirm with emily

File: web/src/pages/capabilities.js (hunk 3 above)
Comment (kcbreuil) [reply web/src/pages/capabilities.js:122]: @asasmith yes

File: web/src/pages/about.js (hunk 2 above)
Comment (kcbreuil) [reply web/src/pages/about.js:37]: in the leadership card @asasmith

Comment (asasmith) [reply web/src/pages/about.js:37]: I don't see that in the designs

File: studio/schemas/documents/leaderCard.js (hunk 1 above)
Comment (kcbreuil) [reply studio/schemas/documents/leaderCard.js:25]: But to answer your question, yes they want their LinkedIn added to the leader card. I am confirming the logic with emily.

File: web/src/pages/about.js (hunk 2 above)
Comment (kcbreuil) [reply web/src/pages/about.js:37]: @asasmith If I remember correctly the cards are to link to their LinkedIn, but i am confirming

File: web/src/components/aboutTextSection.js [hunk 4]
```diff
@@ -4,26 +4,27 @@ import React from 'react';
...
 
 export default function AboutText({ data }) {
-    const { textSectionOne, textSectionTwo } = data;
+    const { _rawLeftColumn, _rawRightColumn, iconTextList } = data;
 
     return (
         <div className="container gap-11 grid grid-cols-1 md:grid-cols-2 my-14">
-            <p>
-                {textSectionOne}
+            <div>
```
Comment (asasmith) [inline web/src/components/aboutTextSection.js:14]: Is this extra div necessary?

**Review by asasmith — CHANGES_REQUESTED** [review]

**Review by asasmith — APPROVED** [review]

**Pull request merged**